tests/all/integration/api/helpers/order/test_edit_order.py

```diff
@@ -91,3 +92,150 @@ def test_ignore_on_order_event_update(client, db, user, jwt):
     db.session.refresh(order)
     assert response.status_code == 200
     assert order.event == order_event
+
+
+def get_simple_custom_form_order(db, user):
+    order = OrderSubFactory(amount=234, status='initializing', user=user)
+    attendee = AttendeeSubFactory(order=order)
+
+    CustomForms(
+        event=attendee.event,
+        form='attendee',
+        field_identifier='jobTitle',
+        type='text',
+        is_included=True,
+        is_required=True,
+    )
+
+    CustomForms(
+        event=attendee.event,
+        form='attendee',
+        field_identifier='taxBusinessInfo',
+        type='text',
+        is_included=True,
+        is_required=True,
+    )
+
+    db.session.commit()
+
+    return str(order.id)
+
+
+def get_complex_custom_form_order(db, user):
+    order = OrderSubFactory(amount=234, status='initializing', user=user)
+    attendee = AttendeeSubFactory(order=order)
+
+    CustomForms(
+        event=attendee.event,
+        form='attendee',
+        field_identifier='whatUniversity',
+        name='what university',
+        type='text',
+        is_complex=True,
+        is_included=True,
+        is_required=True,
+    )
+
+    CustomForms(
+        event=attendee.event,
+        form='attendee',
+        field_identifier='whatCollege',
+        name='what college',
+        type='text',
+        is_complex=True,
+        is_included=True,
+        is_required=True,
+    )
+
+    CustomForms(
+        event=attendee.event,
+        form='attendee',
+        field_identifier='naamBatao',
+        name='naam batao',
+        type='text',
+        is_complex=True,
+        is_included=True,
+        is_required=False,
+    )
+
+    db.session.commit()
+
+    return str(order.id)
+
+
+def test_order_status_to_pending_incomplete_simple_custom_form(client, db, user, jwt):
```

Comment:
Codacy found an issue: [Function name "test_order_status_to_pending_incomplete_simple_custom_form" doesn't conform to '[a-z_][a-z0-9_]{1,50}$' pattern](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=5803712)